nextflow_schema.json

```diff
@@ -38,6 +43,65 @@
                 }
             }
         },
+        "snvphyl_pipeline_parameters": {
+            "title": "SNVPhyl pipeline parameters",
+            "type": "object",
+            "description": "Parameters to configure specific pipeline behaviour.",
+            "default": "",
+            "properties": {
```

Comment:
Could you add min/max for as much of these as possible?

For example, `min_coverage_depth` should be > 0. `snv_abundance_ratio` should be between 0 and 1.

The keywords you can use in a JSON Schema is at https://json-schema.org/understanding-json-schema/reference/numeric#range

Reply:
Added in daef5c42c873b6a5671c80af25dac9e924811c48

Simple to do through the web interface!